client/core/core.go

```diff
@@ -2844,6 +2844,12 @@ func (c *Core) initialize() {
 	wg.Wait()
 	c.connMtx.RLock()
 	c.log.Infof("Successfully connected to %d out of %d DEX servers", len(c.conns), len(accts))
+	for dexName, dc := range c.conns {
+		activeOrders, _ := c.dbOrders(dc)
```

Comment:
non-nil error will load 0 orders, and any subsequent db error will cause a shutdown on dex auth or sooner.

Reply:
Leave a comment?